.github/workflows/tests.yml

```diff
@@ -13,10 +13,11 @@ jobs:
         os: [ubuntu-latest, windows-latest, macos-latest]
 
     steps:
-    - uses: actions/checkout@v1
+    - uses: actions/checkout@v2.3.4
     - name: Use Node.js ${{ matrix.node-version }}
-      uses: actions/setup-node@v1
+      uses: actions/setup-node@v2.3.0
```

Comment:
I think `v2` is better than `v2.x.x`.

Reply:
It's done, the workflow is working fine on my repository.